crates/bevy_ui/src/render/mod.rs

```diff
@@ -82,9 +83,6 @@ pub fn build_ui_render(app: &mut App) {
                 extract_default_ui_camera_view::<Camera2d>,
                 extract_default_ui_camera_view::<Camera3d>,
                 extract_uinodes.in_set(RenderUiSystem::ExtractNode),
-                extract_atlas_uinodes
-                    .in_set(RenderUiSystem::ExtractAtlasNode)
-                    .after(RenderUiSystem::ExtractNode),
```

Comment:
It seems like removing `extract_atlas_uinodes` here also means that the ordering constraints referencing `RenderUiSystem::ExtractAtlasNode` aren't doing anything since nothing is in that set now. If its no longer needed we should remove it.

Reply:
Could you make an issue for this?